projects/plugins/jetpack/extensions/blocks/tiled-gallery/tiled-gallery.php

```diff
@@ -180,4 +180,4 @@ private static function is_squareish_layout( $attr ) {
 	}
 }
 
-Tiled_Gallery::register();
+add_action( 'init', array( 'Automattic\\Jetpack\\Extensions\\Tiled_Gallery', 'register' ) );
```

Comment:
One of my pet peeves 🙂
```suggestion
add_action( 'init', array( Tiled_Gallery::class, 'register' ) );
```

Reply:
Changed in f27ccf5339f7526351092a682b9e67b2630ae7e2 😄